sweep/tx_input_set.go

```diff
@@ -121,25 +117,15 @@ type txInputSet struct {
 	wallet Wallet
 }
 
-func dustLimit(relayFee chainfee.SatPerKWeight) btcutil.Amount {
```

Comment:
Main question is if this affects users running with different `min relay fee` settings.  If the dust limit scales based on this, then it may not be possible for them to open channels as implementations may reject what they deem the dust limit.

Reply:
As discussed offline, I think it's somewhat safe to assume that users aren't changing this config value, as otherwise, they would find that their transactions wouldn't always propagate due to to standardness issues.